src/pages/Home.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <main>
     <header>
-      <div class="logo"></div>
+      <img class="logo" src="@images/logo.png" alt="leveldb-inspector logo" />
     </header>
     <form @submit.prevent="connect">
       <p>Select path to LevelDB</p>
-      <label class="file db-path">
-        <input type="text" id="file" webkitdirectory />
-        <span class="file-custom" @click="selectPath">{{ path }}</span>
-      </label>
+      <div class="file" tabindex="0" @click="selectPath" @keyup.enter="selectPath">
```

Comment:
This seems to be problematic. I can't open a database anymore. Please check

Reply:
@emanlodovice Missed the missing `type="button"` in the Choose button below. Should be fixed now, please check again 👍